wca-regulations.md

```diff
@@ -208,8 +208,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 7f1d) Surface: The flat surface on which the Stackmat has been placed. The mat is considered a part of the surface. The timer is not considered a part of the surface.
     - 7f2) The Stackmat timer must be attached to the mat and placed on the surface, with the timer on the side of the mat nearest to the competitor.
         - 7f2a) Exception: For Solving With Feet, the Stackmat must be placed directly on the floor. The timer device may be placed on the side of the mat farthest from the competitor.
-- 7h) The competition area must have a Competitors Area.
-    - 7h1) The organization team may require that a competitor who has been called to compete must remain within the Competitors Area until the competitor has finished all attempts for the round.
+- 7h) The competition area must have a Competitor Waiting Area.
+    - 7h1) The organization team may require that a competitor who has been called to compete must remain within the Competitor Waiting Area until the competitor has finished all attempts for the round.
```

Comment:
Is there any case when the organization team would *not* require this?
Perhaps the text could be merged with 7h:

> The competition area must have a Competitor Waiting Area, in which competitors who are called to compete must remain until they have finished all attempts for the round.

This would also be consistent with A2a below.

Reply:
> Is there any case when the organization team would not require this?

Yes, for rounds with individually schedule attempts (e.g. FMC/multi BLD). And competitors may also want to go to the bathroom, which is usually not a problem for the flow of the competition.

Do you have a suggestion for how to improve this, taking such details into account?

Reply:
Ah, indeed. Here's an attempt:

7h) The competition area must have a [designated] Competitors Waiting Area, in which those who were called to compete must remain until they have finished all attempts for the round.
7h1) The competitor may be allowed to leave the [Competitors] Waiting Area for specific reasons (i.e. going to the bathroom).
7h2) In the case of individually scheduled attempts (see 2u1) it is not necessary that the competitors stay in the [Competitors] Waiting Area.

Reply:
Not a bad start! Could you create a new PR for that?
(Might want to wait until #601 and #602 have settled.)